src/services/formatting/formatting.ts

```diff
@@ -445,7 +445,7 @@ namespace ts.formatting {
                     if ((<MethodDeclaration>node).asteriskToken) {
                         return SyntaxKind.AsteriskToken;
                     }
-                    // fall-through
+                // fall-through

```

Comment:
Yeah I think that would be ideal. If I recall correctly, the Roslyn perspective on indentation would say that this comment doesn't "belong" to the case statement.